hs_collection_resource/receivers.py

```diff
@@ -21,13 +21,11 @@ def pre_check_bag_flag_handler(sender, **kwargs):
     if collection_res_obj.update_text_file.lower() == 'true':
         update_collection_list_csv(collection_res_obj)
         set_dirty_bag_flag(collection_res_obj)
-        collection_res_obj.extra_data = {'update_text_file': 'False'}
-        collection_res_obj.save()
+        collection_res_obj.set_update_text_file(flag='True')
```

Comment:
Shouldn't we update the flag to 'False' in this case?
```suggestion
        collection_res_obj.set_update_text_file(flag='False')
```